web/package.json

```diff
@@ -7,6 +7,7 @@
     "build": "gatsby build",
     "deploy": "npm run now-deploy",
     "develop": "gatsby develop",
+    "empty-cache": "rimraf .cache",
```

Comment:
```suggestion
    "empty-cache": "gatsby clean",
```

Reply:
This probably eliminates `rimraf` as a necessary dependency.

Reply:
I agree! Removed `rimraf` from the project, seems cleaner!